packages/gatsby/src/commands/build.js

```diff
@@ -5,7 +5,7 @@ const buildHTML = require(`./build-html`)
 const buildProductionBundle = require(`./build-javascript`)
 const bootstrap = require(`../bootstrap`)
 const apiRunnerNode = require(`../utils/api-runner-node`)
-const { copyStaticDir } = require(`../utils/get-static-dir`)
+const { copyStaticDirs } = require(`../utils/get-static-dir`)
```

Comment:
@pieh Can we reasonably expect that no one is using this export? It's technically require'able in the gatsby npm module.

Reply:
Those are considered internals. Anything that would end up being `gatsby/dist/something` imports should be considered private APIs (with exception of `commands` directory, which are needed for `gatsby-cli` to not have any changes).

So export name change is fine to make (if it makes sense)

Reply:
it makes sense. It's now copying all static/ dirs from themes and the user's site into the build folder.